model/Hardware/Properties/organizationalEntity.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# organizationalEntity
+
+## Summary
+
+Product Manufacture Name and related information
+
+## Description
+
+This is the name of the OEM organization for a product.
```

Comment:
Shouldn't this be a Relationship between a Product and an Organization ?

The description is not clear what the relationship is: is it the org that conceived, designed, manufactured, sells, buys, uses, ... the product?

Reply:
In the /Core/Artifact is "originatedBy" property. This is a data property. 
What is the forward looking standard we should use in relation to property and relationships?

Update Definition-- please advise